spec/factories/company_values.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :company_value do
+    title { 'test' }
```

Comment:
Try using more descriptive values, so it is easier to find it in the tests - if you will name all fields of different models as 'test' it will be hard to get which is which
```suggestion
    title { 'Company Value Title' }
```